packages/ui/primitives/signature-pad/signature-pad.tsx

```diff
@@ -223,14 +231,23 @@ export const SignaturePad = ({
         {...props}
       />
 
-      <div className="absolute bottom-4 right-4">
+      <div className="absolute bottom-4 right-4 flex gap-2">
         <button
           type="button"
-          className="focus-visible:ring-ring ring-offset-background rounded-full p-0 text-xs text-slate-500 focus-visible:outline-none focus-visible:ring-2"
+          className="bg-primary hover:bg-secondary rounded-full px-3 py-1 text-xs text-slate-500 transition duration-300 hover:text-white focus:outline-none"
           onClick={() => onClearClick()}
         >
           Clear Signature
         </button>
+        {lines.length > 0 && (
+          <button
+            type="button"
+            className="bg-secondary hover:bg-secondary-foreground rounded-full px-3 py-1 text-xs text-white transition duration-300 hover:text-slate-500 focus:outline-none"
+            onClick={() => onUndoClick()}
+          >
+            Undo
+          </button>
+        )}
```

Comment:
This should go on the left hand side of the `Clear Signature` button to stop layout shifting.

Reply:
noted.. making a new commit